markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+  \item Title: Of the following form
+    \begin{lstlisting}
+      Reproducibility package for TITEL(paper)
+    \end{lstlisting}
+  \item Subtitle:
+    \begin{itemize}
+      \item WP: World Bank Policy Research Working Paper (PRWP) number
+      \item PP: DOI number (no https)
+    \end{itemize}
+\end{enumerate}
+
+## Authors and contributors
+Sufficient are
+\begin{enumerate}
+    \item Authoring entity: author, affiliation and ORCID number and email when available; For world bank authors use \textbf{World Bank}. Example:
+      \begin{verbatim}
+        Christoph Lakner World Bank ORCID 0000, clakner@worldbank.org
+      \end{verbatim}
+    \item Contributors: like RAs (not conference participants). Also coeditors if specifically mentioned. FORMAT
+    \begin{lstlisting}
+      Coeditor
+    \end{lstlisting}
+    \item Sponsors: All financial contributions and individual/general grants
+    \item Acknowledgements: Usually empty; Example: Corporate Taxation: special mentioning ministry employees for data assistance (could go to acknowledgment statement)
+    \item Acknowledgement Statement:
+      \begin{verbatim}
+        We are grateful to Benu Bidani, Francisco Ferreira, Luis-Felipe..
+      \end{verbatim}
+      \begin{verbatim}
+        .. and seminar participants at the World Bank’s Poverty ...
+      \end{verbatim}
+  \end{enumerate}
+
+## Version statement
+Sufficient are
+\begin{enumerate}
+    \item Project completion date: FORMAT: YEAR-MONTH
+      \begin{itemize}
+        \item PP: Date at which the article was published: Year-Month
+        \item WP: \textcolor{red}{Must be supplied, bc it is needed for the front page
+        of cataloge}
+      \end{itemize}
+    \item Version date: FORMAT: YEAR-MONTH-DAY
+      \begin{itemize}
+        \item PP: empty
+        \item WP: data of publication as research paper in WB Year-Month-Day
+      \end{itemize}
+\end{enumerate}
+
+## Scope and Coverage
+Sufficient are
+\begin{enumerate}
+    \item Abstract: Abstract of the article; \textcolor{red}{Hyphens are a problem. Check all abstracts. Also carriage returns are a problem, direct copy pasting leads to bad results}
+    \item Geographic areas: Format: Name, Code, Type
+    \begin{lstlisting}
+      Lebanon, LBN, ISO-3166 Alpha-3
+    \end{lstlisting}
+    \item Keywords: No keywords if none are given; Yes if given.
+    \item JEL codes:
+      \begin{verbatim}
+        ID D63
+        Name Equity, Justice, Inequality
+        Parent ID (empty)
+        Vocabulary JEL Classifications
+        URL https://www.aeaweb.org/econlit/jelCodes.php?view=jel
+      \end{verbatim}
+\end{enumerate}
+
+## Processes and Output
+The following fields exist
+\begin{enumerate}
+    \item Process: empty for PP, tbd for WP
+    \item Institutional review board: empty
+    \item Approval process: empty
+    \item Reviews and comments:
+      \begin{itemize}
+        \item WP: If available (tbd, 2023-06-22)
+        \item PP: empty
+      \end{itemize}
```

Comment:
it's not clear to me why any of these should be required, can you make a case?

Reply:
Not really. But I also was only dealing with published papers. One would have to ask the people submitting working papers if this were something useful. I personally do not think it is useful, unless we have something like in the previous comment, a process that we describe. For papers and working papers it does not seem to be useful.

Reply:
i don't think it's useful. let's omit for our protocol.